starter-code/routes/passportRouter.js

```diff
@@ -1,17 +1,61 @@
 const express        = require("express");
 const passportRouter = express.Router();
 // Require user model
-
+const User           = require('../models/user');
 // Add bcrypt to encrypt passwords
-
+const bcrypt         = require('bcrypt');
+const bcryptSalt     = 10;
 // Add passport 
+const passport       = require('passport');
+//ensure login
+const ensureLogin = require("connect-ensure-login");
 
+//GET sign up page 
+passportRouter.get('/signup',(req,res,next)=> {
+  res.render('../views/passport/signup.hbs');
```

Comment:
On line 61 of your app.js, you have app.set('views', path.join(__dirname, 'views')) which **automatically tells the app where all the views are stored** so there's no need for you to add '../views/' when using res.render()